src/stories/Library/search-result-item/SearchResultItem.tsx

```diff
@@ -43,27 +45,17 @@ export const SearchResultItem = ({
         <p className="text-small-caption">{`Af ${author} (${year})`}</p>
       </div>
       <div className="search-result-item__availability">
-        <AvailabilityLabel
-          manifestationType="Bog"
-          availability="Hjemme"
-          status="available"
-        />
-        <AvailabilityLabel
-          manifestationType="Ebog"
-          availability="Online"
-          status="available"
-        />
-
-        <AvailabilityLabel
-          manifestationType="Lydbog (cd-mp3)"
-          availability="Udlånt"
-          status="unavailable"
-        />
-        <AvailabilityLabel
-          manifestationType="Lydbog (net)"
-          availability="Online"
-          status="available"
-        />
+        {Array(availabilityLabels)
+          .fill(0)
+          .map(() => {
+            return (
+              <AvailabilityLabel
+                manifestationType="Lydbog (cd-mp3)"
+                availability="Hjemme"
+                status="available"
```

Comment:
Consider adding variations e.g. by making every other type unavailable and using a set of predefined manifestation types.